### ISSUE TITLE
Improve footer 508 compliance

### ISSUE THREAD

**Issue by ajakz** [cires-ncei/onestop#86]
Currently the footer is just recognized as a list of links. Some work should be done to clarify to screen readers that they are in a footer, the links are external to the site, etc.
